pluralsight/design-system
# fix(datepicker): remove unnecessary forced focus

### What You're Solving
closes #2026 

DatePicker was stealing focus from other inputs when sharing a page view. This PR solves that issue.

### How to Verify
- Fire up ol' DatePicker SB and go to the "Example" option which mimicks the docs page
- DatePicker input should allow focus to itself and other inputs sharing the view without causing issues of each working independently and correclty
